docs/README.md

```diff
@@ -0,0 +1,10 @@
+# Onsite-demo documentation
+
+## Description of doc structure
+
+- Features used and project decisions [here](https://github.com/chaordic/onsite-demo/blob/feat/master/docs/features.md).
+- Brief of widget types as product [here](https://github.com/chaordic/onsite-demo/blob/feat/master/docs/widgets-brief.md).
+- Brief of the recommendation doc, description of api response [here](https://github.com/chaordic/onsite-demo/blob/feat/master/docs/rec-description.md).
```

Comment:
Acho melhor não abreviar a palavra `documentation`, sempre que for se referir a ela em texto, escreve a palavra inteira.